lib/client/api.go

```diff
@@ -3581,6 +3589,71 @@ func (tc *TeleportClient) Login(ctx context.Context) (*Key, error) {
 	return key, nil
 }
 
+// AttemptDeviceLogin attempts device authentication for the current device.
+// It expects to receive the latest activated [key], as acquired via
```

Comment:
Does this actually link to the `key` function parameter?

Reply:
It doesn't. :(

Removed.